socket.c

```diff
@@ -67,6 +76,28 @@ static void socket_init_not_established(struct thread *t, int s)
                 if (err)
                         PLOG_ERROR(t->cb, "setsockopt(SO_LINGER)");
         }
+#ifdef WITH_TCPDEVMEM_CUDA
+        if (!t->f_mbuf && opts->tcpd_gpu_pci_addr) {
+                if (tcpd_cuda_setup_alloc(t->opts, &t->f_mbuf, t)) {
+                        LOG_FATAL(t->cb, "%s: failed to setup devmem CUDA socket",
+                                  __func__);
+                        exit(1);
```

Comment:
Is the exit(1) necessary? I'm thinking LOG_FATAL should already stop the process.